src/components/barChart.module.scss

```diff
@@ -0,0 +1,63 @@
+$gray: #e7e7e7;
+$blue: #5fb2fb;
+$spendRed: #fb7b5f;
+
+p {
+  font-size: 2rem;
+}
+
+.bar {
+  border-radius: 0.8rem;
+  height: 100%;
+}
+
+.barBackground {
+  background-color: $gray;
+  border-radius: 0.8rem;
+  height: 1.6rem;
+  overflow: hidden;
+  width: 100%;
+}
+
+.barContainer {
+  align-self: center;
+  flex: 2;
+}
+
+.chart {
+  margin: 2.9rem 0;
+}
+
+.contributions {
+  background-color: $blue;
+}
+
+.expenditures {
+  background-color: $spendRed;
+}
+
+.label {
+  align-self: center;
+  flex: 1;
+}
+
+.row {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  margin-bottom: 2.9rem;
+}
+
+.value {
+  align-self: center;
+  flex: 1;
+  font-weight: bold;
+  margin-right: 1rem;
+  text-align: right;
+}
+
+@media screen and (max-width: 760px) {
+  .barContainer {
+    flex: 0 0 50%;
```

Comment:
I want all the bars to be the same size, and with a smaller screen size this seems to look the best - just explicitly setting to 50% and then letting the text wrap if needed.